Meadowlark-js/packages/meadowlark-core/src/plugin/PluginLoader.ts

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+/* eslint-disable global-require */
+import dotenv from 'dotenv';
+import { MeadowlarkBackendPlugin } from './backend/MeadowlarkBackendPlugin';
+import { NoMeadowlarkBackendPlugin } from './backend/NoMeadowlarkBackendPlugin';
+
+dotenv.config();
+
+const loadedBackendPlugin = process.env.BACKEND_PLUGIN_MODULE
+  ? require(`../packages/${process.env.BACKEND_PLUGIN_MODULE}/index`).initializeBackendPlugin()
+  : NoMeadowlarkBackendPlugin;
+
+export function backendPlugin(): MeadowlarkBackendPlugin {
```

Comment:
When I first read code above that calls `backendPlugin()` I thought it might be building something with every call. I wonder if it would be a little easier to read  with a small rename: `getBackend()` perhaps? Something to think about; I'm not fixated on it.

Reply:
Sounds good. `backendPlugin` was originally a `const` but became a function to make mocking out easy.